src/typeDefs/queries.ts

```diff
@@ -19,6 +19,10 @@ export const queries = gql`
       organizationId: ID!
     ): [ActionItemCategory]
 
+    agendaCategory(id: ID!): AgendaCategory
+
+    agendaCategories: [AgendaCategory]
```

Comment:
No need for this.

Reply:
Hello, do you mean that there is no need for these resolvers or just the typedefs?

Reply:
No need for the resolver.